apps/api/src/app/workflows-v2/shared/build-string-schema.ts

```diff
@@ -0,0 +1,20 @@
+import { JSONSchema } from 'json-schema-to-ts';
+
+/**
+ * Builds a JSON schema object where each variable becomes a string property.
+ */
+export function buildStringSchema(variables: Record<string, unknown>): JSONSchema {
```

Comment:
```suggestion
export function buildJSONSchema(variables: Record<string, unknown>): JSONSchema {
```

The comment above it explains the rest ;)